packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -319,14 +319,14 @@ class _ModalBottomSheetRoute<T> extends PopupRoute<T> {
 ///  * [showBottomSheet] and [ScaffoldState.showBottomSheet], for showing
 ///    non-modal bottom sheets.
 ///  * <https://material.google.com/components/bottom-sheets.html#bottom-sheets-modal-bottom-sheets>
-Future<T> showModalBottomSheet<T>({
+Future<void> showModalBottomSheet({
```

Comment:
this is probably a breaking change, so we probably don't want to change that.

Reply:
It is a breaking change but I usually never see type parameters on method like this one. So I wouldn't expect too much problem.

BTW changing `Future<Null>` to `Future<void>` is also a breaking change as `Future<void>` is not assignable to `Future<Null>` (throws at runtime).